android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -65,11 +65,11 @@ public void setAdFormat(final String value)
             return;
         }
 
-        if ( "banner".equals( value ) )
+        if ( MaxAdFormat.BANNER.getLabel().equals( value ) )
```

Comment:
`MaxAdFormat.BANNER.getLabel()` returns "BANNER", so you should probably use `isEqualIgnoreCase()`.

Reply:
(Unless we're forcing publishers to use that string)

Reply:
They can't use that string since we now don't expose any raw strings.  So this should be okay.